test/jdk/java/io/SequenceInputStream/TransferTo.java

```diff
@@ -0,0 +1,206 @@
+            OutputStreamProvider outputStreamProvider, byte[] inBytes) throws Exception {
+        checkTransferredContents(inputStreamProvider,
+                outputStreamProvider, inBytes, 0, 0);
+    }
+
+    /*
+     * Asserts that the transferred content is correct, i. e. compares the bytes
+     * actually transferred to those expected. The positions of the input and
+     * output streams before the transfer are provided by the caller.
+     */
+    private static void checkTransferredContents(InputStreamProvider inputStreamProvider,
+            OutputStreamProvider outputStreamProvider, byte[] inBytes, int posIn,
+            int posOut) throws Exception {
+        AtomicReference<Supplier<byte[]>> recorder = new AtomicReference<>();
+        try (InputStream in = inputStreamProvider.input(inBytes);
+            OutputStream out = outputStreamProvider.output(recorder::set)) {
+            // skip bytes until starting position
+            in.skipNBytes(posIn);
+            out.write(new byte[posOut]);
+
+            long reported = in.transferTo(out);
+            int count = inBytes.length - posIn;
+
+            assertEquals(reported, count,
+                    format("reported %d bytes but should report %d", reported, count));
+
+            byte[] outBytes = recorder.get().get();
+            assertTrue(Arrays.equals(inBytes, posIn, posIn + count,
+                    outBytes, posOut, posOut + count),
+                    format("inBytes.length=%d, outBytes.length=%d", count, outBytes.length));
+        }
+    }
+
+    /*
+     * Creates an array of random size (between min and min + maxRandomAdditive)
+     * filled with random bytes
+     */
+    private static byte[] createRandomBytes(int min, int maxRandomAdditive) {
+        byte[] bytes = new byte[min +
+                                (maxRandomAdditive == 0 ? 0 : RND.nextInt(maxRandomAdditive))];
+        RND.nextBytes(bytes);
+        return bytes;
+    }
+
+    private interface InputStreamProvider {
+        InputStream input(byte... bytes) throws Exception;
+    }
+
+    private interface OutputStreamProvider {
+        OutputStream output(Consumer<Supplier<byte[]>> spy) throws Exception;
+    }
+
+    private static InputStreamProvider byteArrayInput() {
+        return bytes -> {
+            InputStream is1 = new ByteArrayInputStream(bytes, 0, bytes.length / 2);
+            InputStream is2 = new ByteArrayInputStream(bytes, bytes.length / 2, bytes.length);
+            return new SequenceInputStream(is1, is2);
+        };
+    }
+
+    private static OutputStreamProvider byteArrayOutput() {
+        return new OutputStreamProvider() {
+            @Override
+            public OutputStream output(Consumer<Supplier<byte[]>> spy) {
+                ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+                spy.accept(outputStream::toByteArray);
+                return outputStream;
+            }
+        };
+    }
+
+}
```

Comment:
As auto-detected, there is no newline at the end of this file. Otherwise it appears all right.

Reply:
@mkarg Please take account of the JDK 20 rampdown at 16:00 UTC Thursday and that Pacific Time is eight hours behind. Thanks.

Reply:
@bplb Thank you for your review. I have added the missing EOL and would be happy if you would mark this PR as reviewed, so I can integrate it into JDK 20. Thanks! :-)

Reply:
@bplb @AlanBateman All checks have passes. As time runs out, can you please set the review mark in Github? Thanks! :-)